pylossless/pipeline.py

```diff
@@ -439,6 +429,17 @@ def __init__(self, config_fname):
     def load_config(self):
         self.config = read_config(self.config_fname)
 
+    def set_montage(self, raw):
+        chan_locs = self.config['project']['analysis_montage']
+        if chan_locs in mne.channels.montage.get_builtin_montages():
+            # If chanlocs is a string of one the standard MNE montages
+            montage = mne.channels.make_standard_montage(chan_locs)
+        else:  # If the montage is a filepath of a custom montage
+            raise ValueError('Montage should be one of the default MNE montages as'
+                             ' specified by mne.channels.get_builtin_montages()')
+            # montage = read_custom_montage(chan_locs)
+        raw.set_montage(montage, **self.config['project']['set_montage_kwargs'])
+
```

Comment:
@christian-oreilly 

`mne.make_fixed_length_epochs` might be able to replace our own epoching function?

Reply:
That's cool. I was missing something like that. I guess I am a bit lagging behind as this feature has been released in March 2020! Yes, we would need to test it first, but it looks like it would.

Regarding your correlation thing, MNE guys spent a significant amount of thoughts on covariance matrix estimation (https://mne.tools/stable/generated/mne.compute_covariance.html#mne.compute_covariance) since it is required for source estimation. I remember reading a paper from Alex Gramfort on that (https://pubmed.ncbi.nlm.nih.gov/25541187/). This MNE function has fancier approaches to estimate such covariance more reliably than vanilla covariance estimators. Covriance is closely linked to correlation (you can think of it as an unnormalized correlation) so I imagine this can definitely be used (and probably should). 

I would also definitely consider using `compute_bridged_electrodes`. Or what about adding a flag in the config file which can select between using the legacy approach ("chan_var") or the MNE/Tenke-Kayser approach (https://www.sciencedirect.com/science/article/abs/pii/S1388245700005538?via%3Dihub)? If we feel so inclined, we could benchmark these two alternative approaches, e.g., in a pylossless paper.

Reply:
Yeah I am also inclined to use these available mne functions rather than re create the wheel. Like you said, a lot of thought was put into these methods.

Sounds good let's chat more on Friday! I'll merge this PR.